DataFormats/TauReco/src/PFTau3ProngSummary.cc

```diff
@@ -10,10 +10,10 @@ PFTau3ProngSummary::PFTau3ProngSummary(reco::PFTauTransverseImpactParameterRef T
     has3ProngSolution_.push_back(false);
     solution_Chi2_.push_back(0);
     thetaGJsig_.push_back(0);
-    tau_p4_.push_back(TLorentzVector(0,0,0,0));
-    daughter_PDGID_.push_back(std::vector<int>());
-    daughter_charge_.push_back(std::vector<int>());
-    daughter_p4_.push_back(std::vector<TLorentzVector>());
+    tau_p4_.emplace_back(0,0,0,0);
```

Comment:
it may be still more readable to keep the type in place for cases with non-trivial constructors